### PR TITLE
Fix case is not insensitive

### PR DESCRIPTION
**Pull request by Endermanbugzjfc**

### CODE REVIEW

File: tests/CaseInsensitiveTest.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+declare(strict_types=1);
+
+namespace SOFe\InfoAPI;
+
+use PHPUnit\Framework\TestCase;
+
+final class CaseInsensitiveTest extends TestCase {
+
+	private function provideInfo(InfoAPI $api) : void {
```
Comment (SOF3) [inline tests/CaseInsensitiveTest.php:29]: Methods should always be static if it doesn't use `$this`.

**Review by SOF3 — CHANGES_REQUESTED** [review]
Also add tests for infos with upper and mixed case names

File: tests/CaseInsensitiveTest.php (hunk 1 above)
Comment (Endermanbugzjfc) [reply tests/CaseInsensitiveTest.php:29]:
```suggestion
	private static function provideInfo(InfoAPI $api) : void {
```

**Comment by SOF3** [comment]
I don't think ChildName is the correct place to fix this. Only the actual pathfinding logic should be affected.

**Review by SOF3 — APPROVED** [review]

**Pull request merged**